### ISSUE TITLE
v3.16.0 no longer connects. Downgrading to v3.15.5 and I can connect again.

### ISSUE THREAD

**Issue by jbardi** [NordSecurity/nordvpn-linux#11]
I have tried uninstalling and reinstalling several times to ensure it was not a fluke, but the results are always the same. I am able to connect without any problems with "nordvpn c" command on v3.15.5, but if I upgrade to v3.16.0, any attempt to connect with "nordvpn c" or even a specific city results in the following error immediately upon hitting enter:

Whoops! Connection failed. Please try again. If the problem persists, contact our customer support.

I have rebooted, logged out and back in, even completely purged nordvpn and reinstalled, but am unable to connect without that error on v3.16.0, however, every time I downgrade back to v3.15.5, it connects just fine.

I'm running Ubuntu Jammy amd64

**Comment by bjlockie** [comment]
Are you using wireguard?
If yes, try the openvpn protocol.

**Comment by diegoortizmatajira** [comment]
Hi, I can confirm the same error message
```
2023/03/28 09:50:29 [Error] TELIO: [repeated 100 times!] libtelio-build/libtelio/crates/telio-wg/src/adapter/linux_native_wg.rs:102 LinuxNativeWg: [GET01] Unable to get interface from WireGuard. Make sure it exists and you have permissions to access it.
```
After switching technology to OpenVpn it work, so the issue is only happening with NordLynx.

**Comment by djpeebles** [comment]
I've just updated to 3.16.1 and get the same error when trying to connect using nordlynx.  Changing the technology setting to openvpn solves the problem.  Going back to the previous version 3.15.5-1 there is no problem with nordlynx.

**Comment by MatiasDuhalde** [comment]
Hello, I also updated recently and I'm getting the same error:
```
Whoops! We couldn't connect you to 'France'. Please try again. If the problem persists, contact our customer support.
```

And in the journal:
```
nordvpnd[2762]: 2023/04/08 16:27:03 [Error] TELIO: libtelio-build/libtelio/crates/telio-wg/src/adapter/linux_native_wg.rs:102 LinuxNativeWg: [GET01] Unable to get interface from WireGuard. Make sure it exists and you have permissions to access it.
```

I'm running the latest Arch Linux (6.2.10-arch1-1)

**Comment by bjlockie** [comment]
@MatiasDuhalde 
Does OpenVPN work for you?
`sudo nordvpn set tehnology openvpn`
I don't know what the problem is but hopefully they look at it.

Try to reinstall the wireguard packages?

**Comment by MatiasDuhalde** [comment]
Hello @bjlockie :wave: 

Yes, I can connect without problems if I switch the technology from NordLynx to OpenVPN.

I did reinstall WireGuard tools through my package manager but it didn't seem to have an effect. `wg --version` is currently showing :
```
wireguard-tools v1.0.20210914 - https://git.zx2c4.com/wireguard-tools/
```

**Comment by kelvinchin12070811** [comment]
I have the same issue also on Raspberry Pi 4b 4GB (Raspberry Pi OS on Debian GNU/Linux 11 bullseye) and my kernel version reported by neofetch is kernel version 6.1.21-v8+. Downgrading to version 3.15.5 did work but I don't feel it is a permanent solution as your's mesh net feature becomes more and more useful. Btw, if switched to OpenVPN protocol and everything works again.

Below is the log form `sudo journalctl -u nordvpnd > ./daemonlog.txt` which is also the exact log file that I sent to your customer support (and their solution did not fix the issue).

[nordvpn-logs.txt](https://github.com/NordSecurity/nordvpn-linux/files/11241409/nordvpn-logs.txt)

**Comment by dochego** [comment]
Had the same issue with 3.16.0 and 3.16.1 (no connection with NordLynx, Openvpn worked OK). 
It's working again in 3.16.2 for me, so this seems to be fixed.
Linux, Arch, using the nordvpn-bin package.

**Comment by bartoszWojciechO** [comment]
@jbardi, the issue should be resolved in version `3.16.2`. Are you experiencing the same problems after updating?

**Comment by bartoszWojciechO** [comment]
Closing as this issue should be fixed in `3.16.2`. Will reopen if need be.
